sideboard/tests/test_configuration.py

```diff
@@ -74,7 +74,7 @@ def config_overrides_set(self, monkeypatch):
     @pytest.fixture
     def plugin_dirs(self):
         module_path = '/fake/sideboard/plugins/test-plugin/test_plugin'
-        root_path = os.path.join(os.getcwd(), 'plugins', 'test-plugin')
+        root_path = os.path.join(config['plugins_dir'], 'test-plugin')
```

Comment:
This is the more correct way to do this.  I *think* that the new way will work on the MAGFest Vagrant - I know that the old way wasn't working on the Vagrant environments at my day job.  However, this is something we'll want to confirm; we'll also want to confirm that it works in Travis.